app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -351,16 +360,19 @@ public Survey getSurveyMostRecentlyPublishedVersion(StudyIdentifier studyIdentif
         return new QueryBuilder().setStudy(studyIdentifier).isPublished().setSurvey(guid).isNotDeleted().getOne(true);
     }
     
+    // SCAN
     @Override
     public Survey getSurveyMostRecentlyPublishedVersionByIdentifier(StudyIdentifier studyIdentifier, String identifier) {
         return new QueryBuilder().setStudy(studyIdentifier).setIdentifier(identifier).isPublished().isNotDeleted().getOne(true);
     }
```

Comment:
Better make that a near term 1.1 or 1.2 TODO, if our apps are going to use this.